telegraf/telegraf.hc.conf

```diff
@@ -0,0 +1,94 @@
+# Telegraf Configuration
+#
+# Telegraf is entirely plugin driven. All metrics are gathered from the
+# declared inputs, and sent to the declared outputs.
+#
+# Plugins must be declared in here to be active.
+# To deactivate a plugin, comment out the name and any variables.
+#
+# Use 'telegraf -config telegraf.conf -test' to see what metrics a config
+# file would generate.
+#
+# Environment variables can be used anywhere in this config file, simply surround
+# them with ${}. For strings the variable must be within quotes (ie, "${STR_VAR}"),
+# for numbers and booleans they should be plain (ie, ${INT_VAR}, ${BOOL_VAR})
+
+[agent]
```

Comment:
An `[agent]` configuration is not needed with `--once`

Reply:
This is not necessarily true, as even with `--once`  telegraf will use the agent configuration for the metrics buffer limit for example. You can test it by setting this config and looking at the result:
```
[agent]
  metric_batch_size = 1
  metric_buffer_limit = 2
  debug = true
```